DeDRM_plugin/config.py

```diff
@@ -431,16 +431,19 @@ def export_key(self):
         defaultname = "{0}.{1}".format(keyname, self.keyfile_ext)
         filename = choose_save_file(self, unique_dlg_name,  caption, filters, all_files=False, initial_filename=defaultname)
         if filename:
-            with file(filename, 'wb') as fname:
```

Comment:
This block seems to duplicate #1274

Reply:
Yeah, feel free to close that if this is merged instead. With these changes importing from kindle 1.17.1 is working here in Calibre 5.1.0 and key export works help links launch, etc.